app/javascript/packs/bus-route.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+
+import BusStopLink from './bus-stop-link'
+import Direction from './direction'
+
+const BusRoute = (props) => {
+  const busRoute = props.busRoute
```

Comment:
This no semi-colons feels weird

Reply:
yeah. I'm going to get eslint up sometime sooooon.